### ISSUE TITLE
Quiz-Page: Multiple Answer Options (checkbox)

### ISSUE THREAD

**Issue by gius88** [chingu-voyages/moonshot-chingu-quiz#28]
QUIZ PAGE

As a User I will have 4 answer options on each question

Dependent on #21

**Comment by hrebenarMartin** [comment]
Hello, i'd like to take up on this issue. Would that be possible?

**Comment by cMikolai** [comment]
@hrebenarMartin Thank you for your interest in joining this project! Yes, this issue is still open for someone to grab. However, it is a sub task of another issue: https://github.com/chingu-voyages/moonshot-chingu-quiz/issues/21. Would you be interested in kicking the single quiz pages on overall? :-)

**Comment by hrebenarMartin** [comment]
Yes i'd love to :))

**Comment by cMikolai** [comment]
@hrebenarMartin Amazing!

**Comment by ZumDeWald** [comment]
@hrebenarMartin please be sure to place your PR to the `feature/quiz-page` branch and not Dev as this ties into a larger feature, thanks!

**Comment by cMikolai** [comment]
@hrebenarMartin I just wanted to let you know that, if you have any questions and would like to join us in our Discord channel, you could join the free program here: https://www.chingu.io/plans (Explorer plan)

**Comment by cMikolai** [comment]
@hrebenarMartin Hey there, Hope all is well! I was wondering how it is going with the issue? Is there anything we could take over and help with? It's been quite a while now and it's a main feature that's kinda blocking the rest of the team in the moment. Please, let us know if we can support you in any way or take over the task. Thanks!

**Comment by hrebenarMartin** [comment]
Hello, Firstly i apologize that this takes me this long. I already have the project running locally and i studied the code. But i got busy lately with work and university. But i already reserved this weekend for this project. So if you'll allow, i'll definitelly start this evening and will try to finish it on Saturday so i can submit it.

**Comment by cMikolai** [comment]
@hrebenarMartin Hey there, no worries! Sure, take your time.

**Comment by hrebenarMartin** [comment]
@cMikolai Hello, i have just now created a PR for my issues, 21 & 28. I'd like some review or comment, thank you.
